### DISCUSSION TITLE
💡 add Jellyfin "syncplay" feature

### DISCUSSION THREAD

**Discussion by poulpor** [DonutWare/Fladder#90] [Ideas]
### Summary

SyncPlay allows you to create rooms that other users or clients can join in order to share a common viewing experience. There is no limit on the number of users in a room and you are free to join the same room with the same user from multiple clients as well (from Jellyfin Release v10.6.0 changelog)

### Basic Example

In jellyfin Media Player desktop app or web client, you can create a group with other users of the server and play the same media synced between all of them.

### Drawbacks

I don't know.

**Comment by Muletia** [comment]
Would love this. I've been looking for alternatives to Plex with them slowly removing the "Watch Together" feature from the apps, and have found Jellyfin may fill that gap, however I've found the Jellyfin Media Player to be pretty unstable where it'll just stop playing anything after a time - hence trying Fladder.
I love the look and style and would really love to have the Jellyfin SyncPlay capabilities in Fladder too if possible!
Keep up the awesome work!

**Comment by rohoal1012** [comment]
yes please add this feature to the app please that will be awesome

**Comment by panda4k** [comment]
+1 for this. The only reason I've still got the default app around, is for SyncPlay.

**Comment by spooked03** [comment]
+1 would completely replace the default app for me if this was added.

**Comment by mels0n** [comment]
+1 for syncplay

**Reply by mels0n** [reply]
This would be the only android tv app that provides this functionality and would be a selling point on every post.

**Comment by dzuberi** [comment]
I'd like to show my support for this. I've been using Fladder as my main client but when I want to sync with my phone while walking around I have to use the official client.

**Comment by gonzalovlchz** [comment]
Would be great

**Comment by irican-f** [comment]
Hi guys,

I really like Fladder and I also really want syncplay. I started to work on it, i'm close to having a stable beta version. Here's a demo video (sorry for the background music) :

https://github.com/user-attachments/assets/10045bd7-36a5-4d64-ba36-bf3bc9e0b02a

I'll make the PR when I test the feature on multiple devices / platforms.

**Reply by dratiux** [reply]
Great work 💯👏

**Reply by eriksneff** [reply]
Wow that is a slick demo, 💓it!!

**Reply by desmvu** [reply]
great demo

**Reply by AyushSehrawat** [reply]
this is great 🔥

do share the pr here too when it's ready 😄

**Comment by desmvu** [comment]
I would love to see SyncPlay working in Fladder as well

**Comment by irican-f** [comment]
Here's the PR :

https://github.com/DonutWare/Fladder/pull/735
